src/test/java/madalinapopescu/tests/tema7/loghinTestsCuraHealthcare.java

```diff
@@ -0,0 +1,51 @@
+package madalinapopescu.tests.tema7;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+public class loghinTestsCuraHealthcare extends BaseTest{
```

Comment:
Please rename it to LoginTestsCuraHealthcare. Name of the classes start with capital letter in Java.

Reply:
Am modificat numele